erpnext/stock/doctype/stock_entry/stock_entry.js

```diff
@@ -149,7 +153,7 @@ frappe.ui.form.on('Stock Entry', {
 					});
 					frappe.set_route('Form', 'Material Request', mr.name);
 				});
-			});
+			}, __("Create"));
```

Comment:
I think we should start removing all the "Create" buttons, and put that functionality in the + button, the "Create" should be used only when it needs to happen conditionally.

Reply:
@Alchez yeah makes sense. Let's make a separate task for this & handle it in a separate PR